src/MBrace.Azure/Store/BlobStore.fs

```diff
@@ -14,25 +14,26 @@ open MBrace.Azure.Runtime.Utilities
 
 [<AutoOpen>]
 module private BlobUtils =
-    
-    let rootPath = "/"
-    let rootPathAlt = @"\"
-    let delims = [|'/'; '\\'|]
-
-    let isPathRooted (path : string) =
-        path.StartsWith rootPath || path.StartsWith rootPathAlt
+    let containerDelimiter = '@'
 
-    let normalizePath (path : string) =
-        let normalized = path.Split(delims, StringSplitOptions.RemoveEmptyEntries) |> String.concat "/"
-        "/" + normalized
+    let isWasbPath (path:string) =
+        path
+        |> Seq.filter ((=) containerDelimiter)
+        |> Seq.length = 1
 
-    let ensureRooted (path : string) =
-        if isPathRooted path then path else raise <| FormatException(sprintf "Invalid path %A. Paths should start with '/' or '\\'." path)
+    let splitPath =
```

Comment:
Doh :-) There was some code happening previously between the name and the argument.